css/styles.css

```
@@ -1,20 +1,345 @@
+.dialog-box a {
+  color: #0AB5CD;
+  transition: 0.2s all;
+}
+
+.dialog-box i {
+  font-style: normal;
+  color: #d1d1d1;
+}
+
+.dialog-box a:hover {
+  color: #5ECEDB
+}
+
+.dialog-box .dialog-box__name {
+  position: absolute;
+  left: 16px;
+  top: -28px;
+  font-size: 1.1rem;
+  color: #BA3B1F;
+  padding: 4px 24px;
+  background: #FF9A40;
+  border-radius: 40px;
+}
+
+.input__form {
+  background: white;
+  display: flex;
+  flex-direction: column;
+  padding: 32px;
+  align-items: center;
+}
+
+.form__row {
+  max-width: 450px;
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 32px;
+  justify-content: center;
+}
+
+.form__row h6 {
+  width: 100%;
+  display: block;
+  font-weight: 800;
+  font-size: 1.5rem;
+  margin: 16px auto;
+  color: #845E44;
+  text-align: center;
+}
+
+.form__flex-wrap {
+  margin-top: 16px;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 100%;
+  justify-content: center;
+}
+
+.input__group {
+  display: flex;
+  flex-direction: column;
+}
+
+.input__group label {
+  font-size: 1.2rem;
+  font-weight: bold;
+  margin-bottom: 8px;
+  opacity: 0.7
+}
+
+.form__flex-wrap .input__group label {
+  text-align: center;
+  margin-left: 0px;
+  margin-bottom: 8px;
+  margin-right: 32px;
+}
+
+.input__form i {
+  text-align: center;
+  display: block;
+  font-style: normal;
+  color: #d1d1d1;
+  opacity: 0.9;
+  font-size: 0.9rem;
+  margin: 8px auto;
+}
+
+.input__form>.form__row input {
+  margin: 0px auto;
+}
+
+input {
+  border: 0px solid white;
+  border-radius: 40px;
+  padding: 16px 24px;
+  font-size: 2rem;
+  font-family: inherit;
+  color: inherit;
+  font-weight: bold;
+  transition: 0.2s all;
+  margin: 8px 0px;
+  margin-right: 32px;
+}
+
+input[type=number]:placeholder-shown {
+  background: #f3f3f3;
 }
 
-#turnipTable thead tr:first-child {
-    border: none;
+input[type=number]:not(:placeholder-shown) {
+  background: transparent;
+  color: #0AB5CD;
+}
+
+input[type=number]:focus {
+  outline: none;
+  transform: scale(1.1);
+  background: white;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.04), 0 3px 6px rgba(0, 0, 0, 0.07);
+}
+
+input[type=number]:focus::placeholder {
+  opacity: 0;
+}
+
+input[type=number] {
+  width: 60px;
+  text-align: center;
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+input[type=number] {
+  -moz-appearance: textfield;
+}
+
+.reset-button {
+  font-family: inherit;
+  font-weight: bold;
+  color: #FFBABA;
+  padding: 8px 16px;
+  border-width: 0px;
+  border-radius: 40px;
+  background: transparent;
+  font-size: 1.2rem;
+  transition: 0.2s all;
+  position: relative;
+  margin: 32px auto;
+  opacity: 0.5;
+}
+
+.reset-button:hover {
+  cursor: pointer;
+  opacity: 1;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.04), 0 3px 6px rgba(0, 0, 0, 0.07);
+}
+
+
+.table-wrapper {
+  width: 98%;
+  max-height: 50vh;
+  padding: 16px;
+  margin: 0px auto;
+  box-sizing: border-box;
+  overflow-x: scroll;
```

```suggestion
  overflow-x:  auto;
```

The reason for this is that we will set the table width to 100% and this will handle the scroll bar showing up on wider screens that do not need it.